### ISSUE TITLE
Data table naming and description rules

### ISSUE THREAD

**Issue by jongund** [opena11y/oaa-rules#31]
Summary attribute is deprecated in HTML5 
Figure/Caption also possible technique not listed
Need to check HTML5 accessibility mappings
Develop some test pages

**Comment by A11YAngel** [comment]
Hi Jon, 

I read your #31 issue as well as #15 and #16 issues on table description. I am in full support of caption over summary  attributes for several reasons: 

1) Caption attribute is visible on-screen for all users and is rendered by most screen readers. 
2) Summary attribute should be used minimally because it is not visible on-screen and most screen readers do not render this attribute and should be reserved for complex tables as the last resort. 
3) For complex tables, I recommend descriptions be adjacent to tables as indicated by Access Board or be hidden (off-screen) for screen readers. Could aria-describedby be appropriate in place of summary attribute?
4) As you indicated that summary attribute is deprecated in HTML5. 
5) According to Access Board (http://www.access-board.gov/guidelines-and-standards/communications-and-it/about-the-section-508-standards/guide-to-the-section-508-standards/web-based-intranet-and-internet-information-and-applications-1194-22):

"Is the summary attribute an option?

Although highly recommended by some webpage designers as a way of summarizing the contents of a table, the "summary" attribute of the TABLE tag is not sufficiently supported by major assistive technology manufacturers to warrant recommendation. Therefore, web developers who are interested in summarizing their tables should consider placing their descriptions either adjacent to their tables or in the body of the table, using such tags as the CAPTION tag. In no event should web developers use summarizing tables as an alternative to making the contents of their tables compliant as described above."

My understanding is that Access Board recommends the use of caption attribute over summary attribute. 

Regards, 
Lori

**Comment by jongund** [comment]
Rules now recommend caption, aria-label or aria-labelledby for table captioning; and use summary and aria-describedby for descriptions
